Packs/TheHiveProject/pack_metadata.json

```
@@ -0,0 +1,14 @@
+{
+    "name": "TheHiveProject",
+    "description": "Provides an integration, incident type and layout for use with TheHiveProject.",
+    "support": "community",
+    "currentVersion": "1.0.0",
+    "author": "aburt-demisto",
+    "url": "",
+    "email": "",
+    "created": "2020-08-26T09:03:28Z",
+    "categories": [],
+    "tags": [],
```

Threat Intelligence?
```suggestion
    "tags": ["Threat Intelligence"],
```